addons/cookoff/XEH_postInit.sqf

```diff
@@ -58,3 +58,37 @@
         [FUNC(detonateAmmunition), [_vehicle, _mags, _total], _delay] call CBA_fnc_waitAndExecute;
     };
 }, nil, ["Man","StaticWeapon"]] call CBA_fnc_addClassEventHandler;
+
+if (hasInterface) then {
+    // Plays a sound locally, so that different sounds can be used for various distances
+    [QGVAR(playCookoffSound), {
+        params ["_object", "_sound"];
+
+        if (isNull _object) exitWith {};
+
+        private _distance = _object distance (positionCameraToWorld [0, 0, 0]);
+
+        TRACE_3("",_object,_sound,_maxDistance);
+
+        // 3 classes of distances: close, mid and far, each having different sound files
+        private _classDistance = ([["far", "mid"] select (_distance < 952), "close"] select (_distance < 235));
```

Comment:
double select might be faster but this just looks ugly.

also replace the magic numbers with defines?